### PR TITLE
Move css/vendor-imports/mozilla/mozilla-central-reftests/will-change tests to css/css-will-change/will-change

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

File: css/css-will-change/will-change-stacking-context-001.html [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-    height: 100px;
-  }
-</style>
-
-<div class="indicator box"></div>
-<div class="willchange box">
-  <!-- Because will-change: filter creates a stacking context, this child
-       remains on bottom even though it has a higher z-index than the indicator
-       box. -->
-  <div class="child box"></div>
```
Comment (emilio) [inline css/css-will-change/will-change-stacking-context-001.html:35]: Why are these removed? Are they already in the css-will-change subdir?

File: css/css-will-change/will-change-stacking-context-backdrop-filter-1.html [hunk 2]
```diff
@@ -5,8 +5,9 @@
 <link rel="help" href="https://bugs.chromium.org/p/chromium/issues/detail?id=960953">
 <link rel="help" href="https://drafts.csswg.org/css-will-change/#will-change">
 <link rel="help" href="https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty">
-<link rel="match" href="will-change-stacking-context-002-ref.html">
+<link rel="match" href="green-square-100-by-100-ref.html">
 <style>
+  html, body { margin: 0; padding: 0; }
```
Comment (emilio) [inline css/css-will-change/will-change-stacking-context-backdrop-filter-1.html:10]: Why this change?

File: css/css-will-change/will-change-stacking-context-opacity-2.html [hunk 3]
```diff
@@ -4,8 +4,9 @@
 <link rel="author" title="Philip Rogers" href="mailto:pdr@chromium.org">
 <link rel="help" href="https://bugs.chromium.org/p/chromium/issues/detail?id=1119651">
 <link rel="help" href="https://drafts.csswg.org/css-will-change/#will-change">
-<link rel="match" href="will-change-stacking-context-003-ref.html">
+<link rel="match" href="green-square-100-by-100-ref.html">
 <style>
+  html, body { margin: 0; padding: 0; }
```
Comment (emilio) [inline css/css-will-change/will-change-stacking-context-opacity-2.html:9]: Why this change? Also, only margin: 0 is needed.

**Review by emilio — COMMENTED** [review]
Looks good with questions answered.

File: css/css-will-change/will-change-stacking-context-001.html (hunk 1 above)
Comment (nt1m) [reply css/css-will-change/will-change-stacking-context-001.html:35]: It's testing the same thing as will-change-stacking-context-filter-1.html

File: css/css-will-change/will-change-stacking-context-backdrop-filter-1.html (hunk 2 above)
Comment (nt1m) [reply css/css-will-change/will-change-stacking-context-backdrop-filter-1.html:10]: I've switched to using green-square-100-by-100-ref.html which has these rules (it previously had its own ref file).

green-square-100-by-100-ref.html is used by most of the imported tests.

**Review by emilio — APPROVED** [review]
